### DISCUSSION TITLE
I have a Repository that i didnt create.. how to remove myself or delete the repository

### DISCUSSION THREAD

**Discussion by icehijjawi** [community/community#35346] [Repositories] [answered]
SO for a while now i have been receiving email notification from github, but kept ignoring them thinking the are newsletters and such.

Today i logged in and i found out there is a repository under my account which i never created or never heard of.. and i keep getting notifications because of a critical warning or something.. I just want to delete it or remove myself so i can stop receiving these emails.

Tried the help section but got even more confused.. can anyone check this repository and let me know how did i get attached as a creator when i didnt do such thing at all.. 
https://github.com/icehijjawi/github-slideshow

How is this possible ? when i have never for once tried to create anything here..

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @icehijjawi ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!

**Reply by icehijjawi** [reply]
No worries.. thanks you and at the same time sorry for posting under the wrong flair..

**Comment by airtower-luna** [comment, marked as answer]
The repository looks like it was created by the "GitHub Learning Lab" bot. Learning Lab was closed a while ago, but the courses there generally worked by creating a repository where you'd interact with the bot in issues and PRs.

From the [one issue that exists in the repository](https://github.com/icehijjawi/github-slideshow/issues/1) it was created in summer of 2020. Maybe you tried to use Learning Lab then?

Either way, you should be able to delete it like any other repository, see: https://docs.github.com/en/repositories/creating-and-managing-repositories/deleting-a-repository

**Comment by icehijjawi** [comment]
First of all i want to say thank you @airtower-luna.. am irritated that i didnt see the solution earlier.. i just didnt think to scroll down since the page looked as if it ended with the huge space for reason on my end.. and clicked though all the tebs on the left but couldnt find anything relevant to deleting a repository..

Maybe a sub tab for the "danger zone" can be added to the left tabs to visually and easily direct people to what they are looking for.. or maybe its just weird for me.. anyway.. thank for helping..

**Comment by Zakil11** [comment]
I have problem with my repository delete because I didn't add them. And someone blocked my computer
